### PR TITLE
[Dashboard] Welcome Page

### LINKED ISSUE #3302: [Dashboard] Welcome Page
New users not coming from the website but from some Gitpod button on the internet, need to have some initial context about what Gitpod is. Imagine you are on some Open-Source project and they claim, start coding now with a button, you click it but have never heard of Gitpod.

### Designs

| Welcome | Welcome (Spacing) |
|-|-|
| image | image |

See also [design specs](https://www.figma.com/file/nVU4VuaAz8t5YBM3xGhCLS/Welcome-Page-3304?node-id=1%3A2).

### PR DESCRIPTION
**Pull request by svenefftinge**
Changes the login page to welcome. Also removes the left half if the user has seen our website before.

Fixes #3302

Screenshot 2021-03-29 at 17 45 34

### CODE REVIEW

**Comment by gtsiolis** [comment]
Looking at this now! 👀

File: components/dashboard/src/Login.tsx [hunk 1]
```diff
@@ -48,36 +69,43 @@ export function Login() {
...
+                        <Item icon={collaborate} text="Collaborate With Your Team"/>
+                        <Item icon={fresh} text="Fresh Workspace For Each New Task"/>
                     </div>
                 </div>
             </div>
-        </div>
-        <div id="login-section" className="flex-grow flex lg:w-1/2 w-full">
-            <div id="login-section-column" className="flex-grow max-w-2xl flex flex-col h-100 mx-auto lg:mx-0">
+        </div>: null}
+        <div id="login-section" className={"flex-grow flex w-full" + (showWelcome ? " lg:w-1/2" : "")}>
```
Comment (gtsiolis) [inline components/dashboard/src/Login.tsx:97]: **question**: What do you think of using the same layout as with the login, whit gravity towards center?

| Login | Welcome |
|-|-|
| image | image |

File: components/dashboard/src/Login.tsx [hunk 2]
```diff
@@ -4,15 +4,32 @@
...
 import { iconForAuthProvider, simplifyProviderName } from "./provider-utils";
+import automate from "./images/welcome/automate.svg";
+import code from "./images/welcome/code.svg";
+import collaborate from "./images/welcome/collaborate.svg";
+import customize from "./images/welcome/customize.svg";
+import fresh from "./images/welcome/fresh.svg";
+import prebuild from "./images/welcome/prebuild.svg";
+
+
+
```
Comment (gtsiolis) [inline components/dashboard/src/Login.tsx:21]: **thought**: I will keep a reference of these three blank lines to support our linting approach in the future. 😇

File: components/dashboard/src/Login.tsx [hunk 3]
```diff
@@ -48,36 +69,43 @@ export function Login() {
...
+        <div id="login-section" className={"flex-grow flex w-full" + (showWelcome ? " lg:w-1/2" : "")}>
+            <div id="login-section-column" className={"flex-grow max-w-2xl flex flex-col h-100 mx-auto" + (showWelcome ? " lg:my-0" : "")}>
                 <div className="flex-grow h-100 flex flex-row items-center justify-center" >
                     <div className="rounded-xl px-10 py-10 mx-auto">
                         <div className="mx-auto pb-8">
                             <img src={gitpodIcon} className="h-16 mx-auto" />
                         </div>
                         <div className="mx-auto text-center pb-8 space-y-2">
-                            <h1 className="text-3xl">Log in to Gitpod</h1>
+                            <h1 className="text-3xl">Log in</h1>
```
Comment (gtsiolis) [inline components/dashboard/src/Login.tsx:105]: **question**: When the left column is abent, we don't have a single reference of the Gitpod term, right? It should be fine for now.

**Review by gtsiolis — APPROVED** [review]
Looks welcoming! One alignment question. Feel free to ignore the rest. 🍁

File: components/dashboard/src/Login.tsx (hunk 1 above)
Comment (svenefftinge) [reply components/dashboard/src/Login.tsx:97]: To looks a little cross-eyed, no?

File: components/dashboard/src/Login.tsx (hunk 2 above)
Comment (svenefftinge) [reply components/dashboard/src/Login.tsx:21]: 🤓

File: components/dashboard/src/Login.tsx (hunk 3 above)
Comment (svenefftinge) [reply components/dashboard/src/Login.tsx:105]: Yes, we do not. I will make it conditional.

File: components/dashboard/src/Login.tsx (hunk 1 above)
Comment (gtsiolis) [reply components/dashboard/src/Login.tsx:97]: Both approaches should be fine on small and medium screens. Center alignment (current welcome page) could look a bit off on large and extra large screens. Happy to keep both around and decide later which fits best.

Comment (svenefftinge) [reply components/dashboard/src/Login.tsx:97]: 😅

**Pull request merged**